src/app/components/Icons/ProfileIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+function ProfileIcon(): JSX.Element {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="35"
+      height="35"
+      viewBox="0 0 35 35"
+    >
+      <g data-name="Bottom navigation" transform="translate(-213 -752.099)">
+        <path
+          id="profile_1_"
+          data-name="profile (1)"
```

Comment:
```suggestion
      <g  transform="translate(-213 -752.099)">
        <path

```
Group name, id and data-name can be removed. Not needed